pages/dkp/kommander/1.3/projects/project-network-policies/index.md

```diff
@@ -0,0 +1,116 @@
+
+**Pods** - This option selects specific Pods, in the same namespace as the network policy, which should be allowed as ingress sources or egress destinations.
+
+When you select both, you are selecting specific pods from a specific namespace.
+
+There also are options to select all namespaces and/or all pods.
+
+For example, if you wanted to create a default "deny all incoming traffic" policy for a namespace, you could select a namespace by name, and check the Select All Pods check box. By not defining any specific ingress rules, you explicitly create a "deny all traffic" network policy!
+
+When defining ingress rules using the IP Block method, you define a CIDR and exception conditions. CIDR stands for Classless Inter-Domain Routing and is an IP standard for creating unique network and device identifiers. When grouped together so that they share an initial sequence of bits in their binary representation, the range of addresses creates a CIDR block. The block is identified in an IPv4-like notation including a dotted-decimal address, followed by a slash, then a colon and a number from 0-32, for example, 127.0.26.33:31.
+
+## Egress Rules
+Egress rules use a very similar combination of options to define the outgoing traffic from pods, ranges of pods, or namespaces in a Kommander Project. Port, Protocol and Destination options for egress rules define the outgoing traffic. You may need to define egress rules under **Destination: To** for pods, ranges of pods, or namespaces to coincide with the different ingress rules you create. For example, if there are namespaces or pods expecting incoming TCP traffic on port 8080, then some corresponding namespaces or pods will need to specify outgoing TCP traffic on port 8080.
+# Network Policy Examples
+
+## Ingress: Allow access to API service pods from all namespaces
+
+Suppose you need to create a network policy to allow incoming traffic to API service pods in a specific Kommander Project's namespace from any other pod in any namespace that has the label, service.corp/users-api-role: client. For this example, API service pods are those pods that were created with the Label, service.corp/users/role: API.
+
+You can limit the policy to just incoming traffic from select namespaces by adding an ingress rule with these characteristics:
+
+- Use Port 8080 to receive incoming TCP traffic, and...
+- Refuse traffic from pods unless they are client pods that have a Label, such as `service.corp/users/role: client`, which follows a common microservice architecture pattern, "microservice-tier-role: accessmode".
+
+### Provide the General Information
+1. Select **Projects** from the left-hand navigation bar, then select the **Network Policy** tab.
+2. Type "microsvc-users-API-allow" in the **ID Name** field.
```

Comment:
We will need to switch `API` -> `api` for this to pass validation

Reply:
Done.